src/app/components/JumpTo/README.md

```diff
@@ -0,0 +1,12 @@
+## Description - WORK IN PROGRESS - COMPLETE IN SUBTASK
+
+The JumpTo component serves as an in-page navigation menu, similar to a table of contents, allowing users to quickly jump to specific headings within an article. This component renders an `H2`, a list of section titles as `anchor` links, and groups them within a `navigation` landmark. When a link is actioned, the page scrolls down to the relevant section, identified by matching anchor `ids`.
+
+This component is typically used in articles with multiple headings, enhancing content accessibility by providing quick navigation options....
```

Comment:
"enhancing content accessibility by providing quick navigation options...."  to ""enhancing content findability by providing quick navigation options...." ?

Reply:
@pvaliani @holchris @louisearchibald Same as above - https://jira.dev.bbc.co.uk/browse/WSTEAMA-1428

Reply:
And we will add a link to this comment on the jira ticket?

Reply:
I've added them to the ticket, although we'll likely be branching off this PR anyway as we can't merge this in until the change freeze is lifted.